### PR TITLE
Combining JS pages and handling the null selectors - Js merge

### PR DESCRIPTION
**Pull request by LisaGumbo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
...
+            dog++;
+        };
+
+        console.log("You are a cat person:", cat)
+        console.log("You are a dog person", dog)
+        // if certain answers are selected
+        // then display dog gif
+
+        // otherwise
+        //display cat gif
```
Comment (davidgrieser) [inline index.js]: Pseudo-Code seems to be implemented below, possibly remove this code?

File: index.js [hunk 2]
```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
...
+    //then we ignore so the error is not thrown
+    continueDisplayDog.style.display = "none";
+}
+
+if (continueDisplayLlama !== null) {
+    //then we ignore so the error is not thrown
+    continueDisplayLlama.style.display = "none";
+}
+
+// TODO: Dog and llama too
```
Comment (davidgrieser) [inline index.js]: Looks like you accomplished this task above. so is this TODO done?

File: index.js [hunk 3]
```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
 var continueDisplayLlama = document.getElementById('continueLlama');
 continueDisplayLlama.style.display = "none";
```
Comment (davidgrieser) [inline index.js]: Getting errors on other pages as you are still trying to set styles on elements that don't exist. You should only need the code in the if statements for changing style and not lines 3, 5, and 7.

Screen Shot 2021-08-12 at 11 42 07 AM

**Review by davidgrieser — COMMENTED** [review]
Going the right direction but getting some JavaScript errors on `index.html`. Comments below.

File: index.js (hunk 1 above)
Comment (kaylaswartz) [reply index.js]: removed the pseudocode

File: index.js (hunk 2 above)
Comment (kaylaswartz) [reply index.js]: Removed the TODO pseudocode

File: index.js (hunk 3 above)
Comment (kaylaswartz) [reply index.js]: removed the style changes on lines 3, 5, and 7. Now there are no errors showing up on index.html

**Review by davidgrieser — APPROVED** [review]
Looks Good To Merge (LGTM)

**Comment by davidgrieser** [comment]
This should resolve #3

**Comment by davidgrieser** [comment]
Resolves #3

**Pull request merged**